### DISCUSSION TITLE
make the way the native url title is generated the same as the way the structure url title is generated

### DISCUSSION THREAD

**Discussion by omejelle** [ExpressionEngine/ExpressionEngine#3453] [Feature Requests]
EE generates a url title based upon the title field. Structure does that too, however there are some small differences in the outputted url title. 
For instance: 

EE: bart-brands-has-been-appointed-prof-h.c-landscape-urbanism-at-th-owl

structure: bart-brands-has-been-appointed-prof-hc-landscape-urbanism-at-th-owl

It would be nice to have these behave the same way

**Comment by intoeetive** [comment]
I checked the code and it's identical; also tested the samples you provided and got the same result on EE 7.3 and Structure 6.

If you know extra steps needed to replicate this issue, please update - closing the request otherwise
